test/source/tests/tests/compose.ts

```diff
@@ -385,20 +400,16 @@ const baseQuotingTest = async (composePage: Controllable, textToInclude: string)
   expect(await composePage.read('@input-body')).to.include(textToInclude);
 };
 
-const isRecipientElementsExists = async (controllable: ControllablePage, exists: { to?: string[], cc?: string[] }) => {
-  const containerTo = await controllable.waitAny('@container-to', { visible: false });
-  const containerCc = await controllable.waitAny('@container-cc', { visible: false });
-  const recipientsTo = await containerTo.$$('.recipients span');
-  const recipientsCc = await containerCc.$$('.recipients span');
-  const checkIfRecipientsContains = async (emails: string[] | undefined, recipientElements: ElementHandle<Element>[]) => {
-    if (!emails || !emails.length) {
-      return;
-    }
+const expectRecipientElements = async (controllable: ControllablePage, expected: { to?: string[], cc?: string[], bcc?: string[] }) => {
```

Comment:
For whatever reason, after merging, this kept giving me trouble (tests failing).

I was not exactly able to understand how this function works, so that I refactored it in a way I can understand. But it still doesn't work. I think I made it a little stricter - if you leave out one of to/cc/bcc in the expected emails, but the DOM does have it, it would fail.

Anyway, I'm off to sleep now. When you have a chance, please take a look at this and see why is it failing.

Reply:
Done,
I corrected this function.

There were some issues with finding a container with recipients in it and also I removed `.not` when excepting `.length` property of two arrays